src/workloads/query/1MDocCollection_IntId.yml

```diff
@@ -0,0 +1,110 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of IDHACK on queries with int _id.
+
+KeyWords:
+- IDHACK
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 5
+
+ActorTemplates:
+- TemplateName: IDHACK
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "IDHackQuery"}}
+    Type: CrudActor
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
+          Database: *Database
+          Collection: *Collection
+          Operations:
+          - OperationName: find
+            OperationCommand:
+              Filter: {_id: {^RandomInt: {min: 0, max: 999999}}}
+
+Actors:
+# Clear any pre-existing collection state.
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: LoadIntIDs
+  Type: Loader
+  Threads: 10
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: 1000000
+        BatchSize: 1000
+        Document:
+          _id: {^Inc: {start: -200000, multiplier: 100000}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- ActorFromTemplate:
+    TemplateName: IDHACK
+    TemplateParameters:
+      Name: IDHack32Threads
+      Threads: 32
+      OnlyActiveInPhase: 3
+
+- ActorFromTemplate:
+    TemplateName: IDHACK
+    TemplateParameters:
+      Name: IDHack128Threads
+      Threads: 128
+      OnlyActiveInPhase: 4
+
+- ActorFromTemplate:
+    TemplateName: IDHACK
+    TemplateParameters:
+      Name: IDHack256Threads
+      Threads: 256
+      OnlyActiveInPhase: 5
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone-all-feature-flags
+      - standalone-query-stats
+      - standalone
```

Comment:
could you try running this on some of the sharded variants too? And other similar ones? I'd recommend adding:
```
- standalone-query-stats-small-rate-limit
- shard
- shard-query-stats
- shard-lite
- single-replica
- single-replica-query-stats
- replica
- replica-query-stats
```

Reply:
Ditto for other tests.

Reply:
Here is the patch! https://spruce.mongodb.com/version/649ca3a3d6d80a6571c7642d/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

Reply:
Oops, there were system failures because the file names started with a number: "Keys must be strings and match ^[A-Za-z][A-Za-z0-9\-_]*$"

I renamed the files and put up another patch: https://spruce.mongodb.com/version/649cfeb60ae606211531f90a/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC